### DISCUSSION TITLE
How to show a circle progess indicator when playing from remote url

### DISCUSSION THREAD

**Discussion by AhmadFadilB** [bluefireteam/audioplayers#957] [Q&A]
How can i check if the player is still loading the mp3? i want to show a circular progress indicator while it loads and disable some things.

**Comment by Abdullah-Mazumder** [comment]
is there any way to handle loading state?

**Comment by Gustl22** [comment]
See https://github.com/bluefireteam/audioplayers/issues/361 please :)
